test/multiverse/lib/multiverse/suite.rb

```diff
@@ -532,6 +534,11 @@ def trigger_test_run
       end
 
       load @after_file if @after_file
+      begin
+        ::MiniTest.class_variable_get(:@@after_run).reverse_each(&:call)
+      rescue => e
+        puts "Error generating simplecov output: #{e.inspect}"
```

Comment:
Even though SimpleCov is our only known user of `@@after_run` at the moment, I could see this error being raised for reasons other than SimpleCov. 

Would you mind updating the puts statement to be more generic?